classes/04class/exercises/c04-rds01/Faria-Marcio/db-creation.txt

```diff
@@ -0,0 +1,152 @@
+# Creating DBSubnetGroup
+    $ aws rds create-db-subnet-group --db-subnet-group-name private-rds --db-subnet-group-description "Private RDS subnet" --subnet-ids subnet-0a74bea287f32ea1f subnet-043a4115af79ab94c subnet-09a3dadf61841984c
+
+#Output
+    {
+        "DBSubnetGroup": {
+            "DBSubnetGroupName": "private-rds",
+            "DBSubnetGroupDescription": "Private RDS subnet",
+            "VpcId": "vpc-04386a3405acb18af",
+            "SubnetGroupStatus": "Complete",
+            "Subnets": [
+                {
+                    "SubnetIdentifier": "subnet-043a4115af79ab94c",
+                    "SubnetAvailabilityZone": {
+                        "Name": "ap-southeast-2b"
+                    },
+                    "SubnetStatus": "Active"
+                },
+                {
+                    "SubnetIdentifier": "subnet-09a3dadf61841984c",
+                    "SubnetAvailabilityZone": {
+                        "Name": "ap-southeast-2c"
+                    },
+                    "SubnetStatus": "Active"
+                },
+                {
+                    "SubnetIdentifier": "subnet-0a74bea287f32ea1f",
+                    "SubnetAvailabilityZone": {
+                        "Name": "ap-southeast-2a"
+                    },
+                    "SubnetStatus": "Active"
+                }
+            ],
+            "DBSubnetGroupArn": "arn:aws:rds:ap-southeast-2:148446344264:subgrp:private-rds"
+        }
+    }
+
+# Creating the Database
+    $ aws rds create-db-instance --db-name devopsacademy --db-instance-identifier devopsacademy --allocated-storage 20 --db-instance-class db.t2.micro --engine mysql --master-username devops --master-user-password devopsacademy --vpc-security-group-ids sg-06702c080456e135c --db-subnet-group-name private-rds --no-publicly-accessible
+
+# Output
+    {
+        "DBInstance": {
+            "DBInstanceIdentifier": "devopsacademy",
+            "DBInstanceClass": "db.t2.micro",
+            "Engine": "mysql",
+            "DBInstanceStatus": "creating",
+            "MasterUsername": "devops",
+            "DBName": "devopsacademy",
+            "AllocatedStorage": 20,
+            "PreferredBackupWindow": "12:03-12:33",
+            "BackupRetentionPeriod": 10,
+            "DBSecurityGroups": [],
+            "VpcSecurityGroups": [
+                {
+                    "VpcSecurityGroupId": "sg-06702c080456e135c",

```

Comment:
Can you post details on this SG?

How you allow the EC2 instance to communicate with RDS instance?